core/google/api/core/retry.py

```diff
@@ -77,7 +125,8 @@ def exponential_sleep_generator(
             be greater than 0.
         maximum (float): The maximum about of time to delay.
         multiplier (float): The multiplier applied to the delay.
-        jitter (float): The maximum about of randomness to apply to the delay.
+        max_jitter (float): The maximum about of randomness to apply to the
```

Comment:
Ah, I wish you'd brought this up in the last PR. But I'm happy to make this change here.